core/tests/performance_e2e/README.md

```diff
@@ -0,0 +1,395 @@
+# Gateway Performance Measurement
+
+Measure performance of the Gateway SDK.
+
+## Overview
+
+Sample goals:
+- Ensure correctness under stress.
+- Measure latency through gateway.
+- Measure throughput over time.
+
+There are two modules as part of this sample.  There is a simulator module 
+producing messages and a metrics module consuming messages. The modules are 
+coupled by the message contents and the gateway links. The following is a 
+generic overview of the sample gateway:
+
+![Overview](Overview.png) 
```

Comment:
Yes, I forgot to include moving the pictures with the README in this commit.  Will update.